concent_api/common/tests/test_api_view.py

```diff
@@ -270,11 +276,12 @@ def test_that_api_view_should_not_rollback_changes_from_first_transaction_when_s
                     CONCENT_PUBLIC_KEY
                 ),
                 content_type='application/octet-stream',
+                HTTP_CONCENT_GOLEM_MESSAGES_VERSION=settings.GOLEM_MESSAGES_VERSION,
             )
         _update_timed_out_subtask_correct_response_mock_function.assert_called()
         _create_client_and_raise_http400_error_mock_function.assert_called()
 
-        self.assertEqual(Client.objects.count(), 1)
+        self.assertEqual(Client.objects.count(), 2)
```

Comment:
It was because of mocking function `get_one_or_none` with side effect create client. This function was used one more time so that is the reason. After all changes number of DB queries will be reduced and this will be changed back to previous value